cmd/create/userrole/cmd.go

```diff
@@ -0,0 +1,319 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package userrole
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/spf13/cobra"
+	errors "github.com/zgalor/weberr"
+
+	"github.com/openshift/rosa/cmd/link/userrole"
```

Comment:
You should alias this to avoid confusion, as this current package is _also_ called `userrole`